src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -209,8 +219,26 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
         });
     }
 
-    @wire(getRecordCreateDefaults, { objectApiName: '$dataImportBatchName' })
-    dataImportBatchCreateDefaults;
+    appendTrueFalsePicklistToElement() {
+        this.formSections.forEach(section => {
+            if (section.elements) {
+                section.elements.forEach(element => {
+                    const fieldMapping = GeFormService.getFieldMappingWrapper(element.dataImportFieldMappingDevNames[0]);
+
+                    if (isNotEmpty(fieldMapping)) {
+                        if (isTrueFalsePicklist(fieldMapping)) {
+                            element.picklistOptionsOverride = trueFalsePicklistOptions();
+                        }
+
+                        Object.defineProperty(element, 'showDefaultValueInput', {
```

Comment:
@kenlewis-sfdo @psadie-sfdo wanted to call attention to this one last thing. I'm not sure how much we should do this particular action, but it worked out great for adding a getter to an object without having to spin up a new component to define the getter. Let me know what ya'll think.

Reply:
Super interesting! I think I like it

Reply:
👍🏽